src/Assert.php

```diff
@@ -1328,6 +1358,8 @@ public static function lengthBetween($value, $min, $max, $message = '')
     /**
      * Will also pass if $value is a directory, use Assert::file() instead if you need to be sure it is a file.
      *
+     * @psalm-assert string $value
```

Comment:
file_exists wil also cast to string, so

```php
<?php

class F {
    public function __toString()
    {
        return __FILE__;
    }
}

var_dump(file_exists(new F())); // true
```

Reply:
Hmm, I don't think there's a `stringable` type here. I'll probably have to use something like `@psalm-assert string|object` as a simplistic approach

Reply:
Checked here: we don't need to fix this, as all file-related assertions use `Assert::isFile()`, and `Assert::isFile()` starts with `Assert::string()`, so we're good here :+1: